{
    "type": "bundle",
    "id": "bundle--fcfc8260-468e-428b-98fd-eed197931088",
    "spec_version": "2.0",
    "objects": [
        {
            "type": "relationship",
            "id": "relationship--d7ea83fa-87c7-4d36-96d5-aee554504040",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "created": "2017-05-31T21:33:27.074Z",
            "modified": "2022-05-06T17:47:24.257Z",
            "relationship_type": "uses",
            "description": "Several transportation organizations in Ukraine have suffered from being infected by [Bad Rabbit](https://attack.mitre.org/software/S0005), resulting in some computers becoming encrypted, according to media reports. (Citation: Marc-Etienne M.Lveill October 2017)",
            "source_ref": "malware--2eaa5319-5e1e-4dd7-bbc4-566fced3964a",
            "target_ref": "attack-pattern--63b6942d-8359-4506-bfb3-cf87aa8120ee",
            "external_references": [
                {
                    "source_name": "Marc-Etienne M.Lveill October 2017",
                    "description": "Marc-Etienne M.Lveill 2017, October 24 Bad Rabbit: NotPetya is back with improved ransomware Retrieved. 2019/10/27 ",
                    "url": "https://www.welivesecurity.com/2017/10/24/bad-rabbit-not-petya-back/"
                }
            ],
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ]
        }
    ]
}